WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalError.swift

```diff
@@ -38,13 +41,15 @@ final class CardPresentModalError: CardPresentPaymentsModalViewModel {
          transactionType: CardPresentTransactionType,
          image: UIImage = .paymentErrorImage,
          primaryAction: @escaping () -> Void,
+         secondaryAction: @escaping () -> Void,
```

Comment:
nit: if the secondary button is always to email receipt, how about using a closure name with this clear intention?
```suggestion
         emailReceipt: @escaping () -> Void,
```